source/release-notes/1.5.txt

```diff
@@ -0,0 +1,58 @@
+.. _c2c-release-notes-1.5:
+
+===================================
+Release Notes for mongosync 1.5
+===================================
+
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _1.5.0-c2c-release-notes:
+
+1.5.0 Release
+-------------
+
+**Upcoming**
```

Comment:
We just did a trial release successfully, and everything is on track, so: we expect this release tomorrow, July 25. Thanks!

Reply:
Thanks!

Reply:
Fixed